robottelo/config/virtwho.py

```diff
@@ -0,0 +1,499 @@
+"""Define and instantiate the configuration class for Robottelo."""
+import logging.config
+import os
+from configparser import ConfigParser
+from configparser import NoOptionError
+from configparser import NoSectionError
+
+from robottelo.config import casts
+
+LOGGER = logging.getLogger(__name__)
+SETTINGS_FILE_NAME = 'hypervisor.properties'
```

Comment:
we may rename the file to 'virtwho.properties'

Reply:
Ok, changed.